src/core/ext/transport/chttp2/transport/flow_control.cc

```diff
@@ -212,6 +217,29 @@ double TransportFlowControl::SmoothLogBdp(double value) {
   return pid_controller_.Update(bdp_error, dt > kMaxDt ? kMaxDt : dt);
 }
 
+double
+TransportFlowControl::TargetInitialWindowSizeBasedOnMemoryPressureAndBdp()
+    const {
+  const double bdp = bdp_estimator_.EstimateBdp() * 2.0;
+  const double memory_pressure = memory_owner_->InstantaneousPressure();
+  auto lerp = [](double t, double t_min, double t_max, double a, double b) {
```

Comment:
It might be helpful to explain some of the math here and rationale on some of the constants - same below for memory pressure estimate

Reply:
done